frontend_app_html-document/src/css/index.styl

```
@@ -83,6 +83,7 @@
         &__content
           color darkGrey
       &__version
+      // MB - 21/05/2021 - This bit of code doesn't seem to be used anywhere
```

```suggestion
      // MB - 2021-05-21 - FIXME (#xxx)
      // This bit of code doesn't seem to be used anywhere
```

Can you open a technical dept issue for this and replace xxx by the id of this issue?